Dockerfile

```diff
@@ -2,21 +2,21 @@ FROM ubuntu:16.04
 
 # Install base packages, ansible, nodejs
 RUN apt-get update && apt-get -y install \
-        awscli \
-        curl \
-        dnsutils \
-        gcc \
-        git \
-        jq \
-        software-properties-common \
-        unzip \
-        wget && \
+    awscli \
```

Comment:
The previous spacing wasn't an error - these should be indented as they are effectively arguments to the `apt-get` command.